### PR TITLE
Bug 1228294 - Verify that the duplicates found screen is shown

### PR DESCRIPTION
**Pull request by isabelrios**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f80d65ba33fc3e109995604bedf621c7f4b097f5

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini [hunk 1]
```diff
@@ -75,3 +75,6 @@ dsds = true
 external = true
 # Bug 1182525 - [Aries] Deactivate the DSDS acceptance tests
 skip-if = device == "desktop" || device == "aries"
+
+[test_duplicate_phone_number.py]
+skip-if = device == "desktop"
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini]: I think this test is able to run on mulet.

Comment (isabelrios) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini]: Should I remove it then and leave only [test_duplicate_phone_number.py] ?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d408350995c785cf1604fa145e33e735cf4056ef

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini (hunk 1 above)
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini]: Yes

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b010065e59936ef13381758637eb69db8be40e14

**Pull request merged**